phalcon/db/adapter/pdo/postgresql.zep

```diff
@@ -158,6 +158,11 @@ class Postgresql extends PdoAdapter
 				 * Timestamp
 				 */
 				let definition["type"] = Column::TYPE_TIMESTAMP;
+			}elseif memstr(columnType, "time") {
+				/**
+                                 * Time
+                                 */
```

Comment:
Sorry, missed that. Will fix it. Still working on those unit test btw. Try to push them today.